infrastructure/stacks/lambda-iam-roles/main.tf

```diff
@@ -53,7 +53,14 @@ resource "aws_iam_role_policy" "event_processor_policy" {
         "xray:PutTraceSegments",
         "xray:PutTelemetryRecords",
         "lambda:InvokeFunction",
-        "sqs:*"
+        "sqs:*",
+        "dynamodb:BatchGetItem",
```

Comment:
Can these new permissions be moved to their own statement and locked to the specific dynamodb table with this variable `TF_VAR_change_events_table_name`

Reply:
Sure